.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Please add `node_modules` as part of the files to be ignored by git

Reply:
Hi @AshakeE.. Thank you for your feedback. I've just submitted the PR for your review. Let me know what you think